adr/context.py

```diff
@@ -19,13 +19,14 @@
                     it will not be listed on webapp and cli
 """
 import ast
+import collections
 import inspect
 from copy import deepcopy
 
 from jsone.interpreter import ExpressionEvaluator
 from jsone.prattparser import prefix
 
-
+COMMON_CONTEXTS = collections.OrderedDict()
 COMMON_CONTEXTS = {
```

Comment:
This is just overwriting the OrderedDict with the normal one. You'll need to individually assign each key:

    COMMON_CONTEXTS['attribute'] = [['--at'], ... ]

Alternatively we could turn this into a list of dicts like `RUN_CONTEXT`.

Reply:
Thank you I updated the code.